Source/Core/DolphinQt2/MainWindow.cpp

```diff
@@ -37,7 +40,9 @@
 
 #include "InputCommon/ControllerInterface/ControllerInterface.h"
 
-MainWindow::MainWindow() : QMainWindow(nullptr)
+#include "UICommon/UICommon.h"
+
+MainWindow::MainWindow() : QMainWindow(nullptr), m_stop_requested(false)
```

Comment:
Sorry, I missed this during my previous reviews, but can you remove `m_stop_requested(false)` and instead move the initialisation to the header? (`bool m_stop_requested = false;`)